test/profiles/test2.profile

```diff
@@ -1,6 +1,6 @@
 caps
 seccomp
-  private
-  include test.profile
-  include test.local
-  include test25.profile
+private
+include test.profile
+include test.local
+include test25.profile
```

Comment:
I think this is intentional to test if firejail can parse commands with leading spaces.

Reply:
> I think this is intentional to test if firejail can parse commands with
> leading spaces.

Good catch; fixed on commit bdb4f03ed ("test2.profile: restore leading spaces",
2023-02-25).